ci/docker/deploy-amdgpu.Dockerfile

```
@@ -70,7 +70,7 @@ RUN if [ "$USE_MKL" = "ON" ]; then \
 # Deploy Extra RocBlas files separately.
 ARG USE_ROCBLAS=OFF
 RUN if [ "$USE_ROCBLAS" = "ON" ]; then \
-      cp -r `spack -e ci location -i rocblas`/lib/library ${DEPLOY}/usr/lib ; \
+      cp -r `spack -e ci location -i rocblas`/lib/rocblas/library ${DEPLOY}/usr/lib ; \
```

I ran the tests on ault and primarily gemm tests failed because they couldn't find Tensile (`rocBLAS error: Cannot read /root/DLA-Future.bundle/usr/lib/rocblas/library/TensileLibrary.dat: Illegal`).  I think the following will fix it:
 ```suggestion
      mkdir ${DEPLOY/usr/lib/rocblas
      cp -r `spack -e ci location -i rocblas`/lib/rocblas/library ${DEPLOY}/usr/lib/rocblas ; \
```

---

You need the `mkdir` there for the copy to work.